boards.txt

```diff
@@ -9895,6 +9895,96 @@ m5stack-fire.menu.DebugLevel.verbose.build.code_debug=5
 
 ##############################################################
 
+m5stack-station.name=M5Stack-Station
+
+m5stack-station.bootloader.tool=esptool_py
+m5stack-station.bootloader.tool.default=esptool_py
+
+m5stack-station.upload.tool=esptool_py
+m5stack-station.upload.tool.default=esptool_py
+m5stack-station.upload.tool.network=esp_ota
+
+m5stack-core2.upload.maximum_size=6553600
+m5stack-core2.upload.maximum_data_size=4521984
```

Comment:
Please fix the name of the board to `m5stack-station`

Reply:
Sorry, this has been corrected